lib/core/data/enums/activity_type.dart

```diff
@@ -9,12 +9,12 @@ enum LFGActivityType {
   ///
   /// See [Activity.custom] for more info.
   ///
-  /// All custom activities registers in `/create custom` command.
-  custom,
+  /// All custom activities registers in `/create activity` command.
+  activity,
 
   /// Dungeon activity, stored in `/activities/dungeons.json` file.
   ///
-  /// Main difference from [LFGActivityType.custom] is that it has fixed number of members: 3.
+  /// Main difference from [LFGActivityType.activity] is that it has fixed number of members: 3.
```

Comment:
I think enum name `LFGActivityType.activity` sounds odd. Maybe rename `LFGActivityType` to `LFGType`?